packages/discovery-provider/scripts/prod-server.sh

```diff
@@ -24,24 +24,12 @@ fi
 
 audius_discprov_loglevel=${audius_discprov_loglevel:-info}
 
-if [[ "$audius_openresty_enable" == true ]]; then
```

Comment:
we don't need this anymore?

Reply:
this optionally starts openresty as background process inside of discovery container...
but it's just in different container now.